charts/akeyless-secure-remote-access/templates/horizontalPodAutoscaler.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if eq .Values.sshConfig.enabled true }}
 {{- if .Values.sshConfig.HPA.enabled }}
 {{- if ne (include "checkPersistenceVolume" .) "true" }}
-  {{ fail "ERROR - Cannot enable HPA without defining a shared persistent storage volume" }}
+  {{ printf "WARNING - HPA without defining a shared persistent storage volume can cause unexpected behavior with multiple pods" }}
```

Comment:
HPA without defining a shared persistent storage volume can not support multiple pods